qcodes/instrument_drivers/oxford/triton.py

```diff
@@ -7,148 +7,222 @@
 from typing import Any, Optional, Union
 
 from qcodes.instrument import IPInstrument
-from qcodes.validators import Enum, Ints
+from qcodes.validators import Enum, Ints, Numbers
 
 
 class OxfordTriton(IPInstrument):
     r"""
     Triton Driver
 
     Args:
-        tmpfile: Optional: an exported windows registry file from the registry
+        name (str): name of the cryostat.
+        address (str, optional): IP-address of the fridge computer. Defaults to None.
+        port (int, optional): port of the oxford program running on the fridge computer.
+            The relevant port can be found in the manual for the fridge
+            or looked up on the fridge computer. Defaults to None.
+        terminator (str): Defaults to '\r\n'
+        tmpfile (str, optional): an exported windows registry file from the registry
```

Comment:
Lets remove the types from the docstrings. They are already in the signature and these are likely to fall out of sync

Reply:
The types are now removed from the docstring